vaas/vaas/router/models.py

```diff
@@ -6,6 +6,25 @@
 from vaas.cluster.models import LogicalCluster
 from vaas.manager.models import Director
 
+RESPONSE_STATUS_CHICES = (
+    ('301', 'Move Pernamentlly'),
+    ('302', 'Found'),
+    ('307', 'Temporary Redirect')
+)
+
+class Rewrite(models.Model):
+    condition = models.CharField(max_length=512)
```

Comment:
it shouldn't be a field made from three other forms? in our MVP we decide to made condition from method+domain+path
![Zrzut ekranu 2022-11-21 o 08 10 38](https://user-images.githubusercontent.com/12686417/202987510-411fac75-9cb7-4b82-8bdb-8b50e61f44ea.png)

Reply:
Yes you are right. We will combine condition from this three field exposed in form. 
We have same field in Route:
```python
class Route(models.Model):
    condition = models.CharField(max_length=512)
    ...
```